locust/static/locust.js

```diff
@@ -170,19 +170,45 @@ $(".edit_config_link").click(function(event) {
     
 });
 
-$('#upload_btn_submit').click(function(event){
+$("#directories .select2").select2({
+    placeholder: "Select a state"
+});
+
+
+$('#upload_py_submit').click(function(event){
     event.preventDefault();
-    $('#upload_file_form').submit();
+    $('.upload_file_form_test_file').submit();
 });
 
-$("#directories .select2").select2({
-    placeholder: "Select a state"
+$('#upload_json_submit').click(function(event){
+    event.preventDefault();
+    $('.upload_file_form_json').submit();
 });
 
-$('#upload_file_form').submit(function(event) {
+$('.upload_file_form_test_file').submit(function(event) {
     event.preventDefault();
-    var form = $('#upload_file_form')[0];
-    var form_data = new FormData(form);
+    var form_data = new FormData($('.upload_file_form_test_file')[0]);
+    $.ajax({
```

Comment:
how about put this ajax to a function? and the function receive form data parameter. for example 
```function call_upload_file(form_data){$.ajax({...})}```
and inside submit handler all you need to do is call that function with passing form data parameter